packages/dd-trace/src/opentracing/span_context.js

```diff
@@ -49,14 +49,14 @@ class DatadogSpanContext {
         ? this._trace.tags[TRACE_ID_128] + this._traceId.toString(16).padStart(16, '0')
         : this._traceId.toString(16).padStart(32, '0')
     }
-    return this._traceId.toString(10)
+    return this._traceId?.toString(10)
```

Comment:
These should guarantee a string. Also, it should not be possible for IDs to be undefined to begin with, so this is working around a bug. If that's needed for OpenTelemetry, let's move that logic only to OpenTelemetry, or translate to something valid on our side.